docs/projects/templates/project-proposal-pr-template.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Project Proposal
+about: A suggestion for a new project to work on
+labels: "💬 talk: discussion"
+title: "<Replace this with actual title>"
```

Comment:
I don't quite understand the `name` and `about` fields here.
Is the name always "Project Proposal"? If not, what is the difference between `name` and `title`?
What do we write in `about`? A one-sentence short description of the project, a short version of the `description` below?

Reply:
Oh, this is the meta used by GitHub in issue/pr templates. So it's the `name` and `about` that appears in [this list](https://github.com/WordPress/openverse/issues/new/choose):

![CleanShot 2023-02-08 at 08 19 57](https://user-images.githubusercontent.com/6351754/217541544-c462328e-a4af-4e25-961b-24d6bba566d8.png)

Reply:
Oh, I see! Thank you :)